environments/prod/platform-hmcts-net.yml

```
@@ -603,4 +603,36 @@ cname:
   - name: "_6a1b0148b51aa81b6efa9ee381182f7d.www.ccd"
     ttl: 300
     record: "3781917CFA062884E530D8F4801CAA7C.B379E5FF10F7E956ADEF47D7A55DFA5B.83e6341c952eb1f0fe89.comodoca.com."
-
+  - name: "_94FBE046A53C70650B5F488AE7D5EE4B.splunk.platform.hmcts.net"
```

don't include the full name in the name, the zone will be appended automatically, applies throughout

```suggestion
  - name: "_94FBE046A53C70650B5F488AE7D5EE4B.splunk"
```

---

OK, will edit throughout